tests/page-objects/default/enketo/undo-death-report.page.js

```diff
@@ -0,0 +1,14 @@
+const FORM ='form[data-form-id="undo_death_report"]';
+
+const submitReporConfirmation = (value) => $(`${FORM} ` +
+  `input[name="/undo_death_report/undo/undo_information"][value="${value}"]`);
+
+const selectSubmitReportConfirmation = async (value = 'yes') => {
```

Comment:
If you decide to do the prev comment then you can do this here too. 
```suggestion
const setConfirmUndoDeathOption = async (value = 'yes') => {
```